composer.json

```diff
@@ -32,6 +32,8 @@
         "composer/xdebug-handler": "^1.4|^2.0",
         "danielstjules/stringy": "^3.1",
         "doctrine/inflector": "^2.0",
+        "ergebnis/json-printer": "^3.1",
+        "idiosyncratic/editorconfig": "dev-master",
```

Comment:
I plan to do something similar as https://github.com/phpstan/phpstan-src where we could use this locked to specific commit. But untill then we need a stable tag here, so Rector can be installed

Reply:
Yes, i am working on it. Sure.

Reply:
Thanks :hugs:

Reply:
Tagged version is out